src/resource/user/user.service.ts

```diff
@@ -26,9 +28,26 @@ export class UserService {
     return await this.prisma.user.findMany();
   }
 
+  async findById(id: string) {
+    const user = await this.prisma.user.findUnique({
+      where: { id },
+      include: { links: true },
+    });
+
+    if (isNil(user)) {
+      this.logger.error(`Cannot found user with id ${id}`);
```

Comment:
Same, now that it's an error here, why do we do nothing but documenting on it?

Reply:
[Same here](https://github.com/A-Comosus/comosus-server/pull/13#discussion_r895024447)